server/handler.go

```diff
@@ -1410,6 +1410,153 @@ func (h *jobsInsertHandler) exportToGCSWithObject(ctx context.Context, response
 	return nil
 }
 
+func (h *jobsInsertHandler) copyFromBigQuery(ctx context.Context, r *jobsInsertRequest, srcTable *bigqueryv2.TableReference) (*bigqueryv2.Job, error) {
+	job := r.job
+	startTime := time.Now()
+
+	tmpf, err := os.CreateTemp("", "*.jsonl")
+	if err != nil {
+		return nil, fmt.Errorf("failed to set up temporary file: %w", err)
+	}
```

Comment:
This file should be deleted at the end of the handler

```suggestion
	tmpf, err := os.CreateTemp("", "*.jsonl")
	if err != nil {
		return nil, fmt.Errorf("failed to set up temporary file: %w", err)
	}
	defer os.Remove(tmpf.Name())
```